docs/_layouts/notify-repo.md

```diff
@@ -4,19 +4,15 @@ layout: page
 
 <!-- markdownlint-disable MD041 -->
 
-<p>
-<a href="https://github.com/NHSDigital/{{page.repo-name}}">https://github.com/NHSDigital/{{page.repo-name}}</a>
-</p>
+<dl>
```

Comment:
This is a more compact display of the repo frontmatter variables:
<img width="811" alt="image" src="https://github.com/NHSDigital/nhs-notify/assets/60653100/532f987c-2e67-44f3-abb3-f5e543923f87">